tests/evaluation/evaluation.test.ts

```diff
@@ -165,17 +165,19 @@ describe("evaluateCells", () => {
   test("error in some function calls", () => {
     const model = new Model();
     setCellContent(model, "A1", '=Sum("asdf")');
-    expect(getEvaluatedCell(model, "A1").value).toBe("#ERROR");
-    expect(getCellError(model, "A1")).toBe(
```

Comment:
why these change? That's a lot of diff and forward port breaking when nothing really had to change, no ? `getEvaluatedCell(model, "A1").value` didn't change, and I'm not sure there is a reason to change `getCellError`

Reply:
Indeed